bom-latest/pom.xml

```
@@ -16,6 +16,7 @@
         <workflow-api-plugin.version>2.40</workflow-api-plugin.version>
         <workflow-cps-plugin.version>2.82</workflow-cps-plugin.version>
         <workflow-job-plugin.version>2.39</workflow-job-plugin.version>
+        <workflow-multibranch-plugin.version>2.21</workflow-multibranch-plugin.version>
```

I just released 2.22 which should fix the PCT issues:

```suggestion
        <workflow-multibranch-plugin.version>2.22</workflow-multibranch-plugin.version>
```